charts/capsule-proxy/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 0.4.4
 description: Helm Chart for Capsule Proxy, addon for Capsule, the multi-tenant Operator
 name: capsule-proxy
 type: application
-version: 0.4.5
+version: 0.5.0
```

Comment:
A minor bump here could be good, since we're adding new fields.

Reply:
Don't you use semVer2 for versioining? This is a minor bump.  
0.4.6 would be a bugfix.